conf/app.routes

```diff
@@ -1,3 +1,3 @@
 # microservice specific routes
 
```

Comment:
I'd like to keep the API as restful as possible.  "message-notification" is more RPC and "/movements/departures" is in line with the backend and public APIs.  I know that it's different from the destination stub but I'd like to bring that into line eventually too.